types/three/examples/jsm/nodes/Nodes.d.ts

```diff
@@ -269,7 +269,14 @@ export {
     luminance,
     ColorAdjustmentMethod,
 } from './display/ColorAdjustmentNode.js';
-export { default as ColorSpaceNode, colorSpace, ColorSpaceNodeMethod } from './display/ColorSpaceNode.js';
+export {
+    default as ColorSpaceNode,
+    ColorSpaceNodeMethod,
+    linearTosRGB,
+    sRGBToLinear,
+    linearToColorSpace,
+    colorSpaceToLinear,
+} from './display/ColorSpaceNode.js';
```

Comment:
nit: I like to keep the order of the exports the same as in the source and type exports at the end to make it easy to verify the exports match the source.

```suggestion
export {
    default as ColorSpaceNode,
    linearToColorSpace,
    colorSpaceToLinear,
    linearTosRGB,
    sRGBToLinear,
    ColorSpaceNodeMethod,
} from './display/ColorSpaceNode.js';
```

Reply:
Agreed!